frontend/src/screens/wallet/Receive.tsx

```diff
@@ -116,6 +127,28 @@ export default function Receive() {
         title="Receive"
         description="Create a lightning invoice that can be paid by any bitcoin lightning wallet"
       />
+      {!!channels?.length && (
+        <>
+          {/* If all channels have less than 20% incoming capacity, show a warning */}
+          {channels?.every(
+            (channel) =>
+              channel.remoteBalance <
+                (channel.localBalance + channel.remoteBalance) * 0.2 ||
```

Comment:
I don't think we need to loop through channels, can we use the balances instead? (see `useBalances`)

Reply:
(also applies to send)

Reply:
Done!